### ISSUE TITLE
How to run individual codemods

### ISSUE THREAD

**Issue by cseas** [SyMind/react-router-dom-5-to-6#1]
Please document how to run a specific codemod script in the readme

**Comment by SyMind** [comment]
@cseas Which codemod you want run separately? I want to know the reason.

**Comment by cseas** [comment]
I'm trying to do the migration on a large codebase so want to run the codemods as a series of steps to make the review process easier. Each step of the codemod will be a separate commit, for example, changing all Route `component` props to `element`.

This is so we can review each step of the migration individually. 

The codebase also uses `withRouter` a lot due to class components. The React Router docs recommend creating a [local wrapper][1] for it but this codemod instead changes all `withRouter` and some other imports to the v5 compat package. So I also want to skip some steps of this codemod.

It'd be great to have that flexibility to run the codemod as a series of scripts instead of one big bang run.
- Each step will be small in scope, making review easier.
- We can choose to skip some steps if the codemod result isn't relevant for us.

[1]: https://reactrouter.com/en/main/start/faq#what-happened-to-withrouter-i-need-it

**Comment by SyMind** [comment]
@cseas I published new version `0.0.2`. You can try it.

You can use the following command.

```bash
npx react-router-dom-5-to-6 src --transform=<codemod-script-name>
```

**Comment by cseas** [comment]
That works, thanks a lot!
